Sources/Constraint.swift

```diff
@@ -226,7 +226,7 @@ public struct Constraint: ConstraintDefinition {
   public internal(set) var firstAttribute: NSLayoutAttribute
   public internal(set) var secondAttribute: NSLayoutAttribute
   
-  private var _enabled = true
+  fileprivate var _enabled = true
```

Comment:
Indeed they are being used only in the lexical scope they are declared in. Fixed